dot/parachain/types/validity_attestation.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package parachaintypes
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+// ValidityAttestation is an implicit or explicit attestation to the validity of a parachain
+// candidate.
+type ValidityAttestation scale.VaryingDataType
+
+// Set will set a VaryingDataTypeValue using the underlying VaryingDataType
+func (va *ValidityAttestation) Set(val scale.VaryingDataTypeValue) (err error) {
+	// cast to VaryingDataType to use VaryingDataType.Set method
+	vdt := scale.VaryingDataType(*va)
+	err = vdt.Set(val)
+	if err != nil {
+		return fmt.Errorf("setting value to varying data type: %w", err)
+	}
+	// store original ParentVDT with VaryingDataType that has been set
+	*va = ValidityAttestation(vdt)
+	return nil
+}
+
+// Value returns the value from the underlying VaryingDataType
+func (va *ValidityAttestation) Value() (scale.VaryingDataTypeValue, error) {
+	vdt := scale.VaryingDataType(*va)
+	return vdt.Value()
+}
+
+// Implicit is for Implicit attestation.
+type Implicit ValidatorSignature //skipcq
+
+// Index returns VDT index
+func (Implicit) Index() uint { //skipcq
+	return 1
+}
```

Comment:
Why is the linter being silenced here?

Reply:
Thanks for pointing it out. I will remove it.